src/main/java/org/broadinstitute/hellbender/dev/pipelines/bqsr/BaseRecalibratorDataflowUtils.java

```diff
@@ -313,12 +353,22 @@ public void finishBundle(DoFn<KV<String, CoGbkResult>, RecalibrationTables>.Cont
             .named("finalizeRecalTables")
             .of(new DoFn<RecalibrationTables, RecalibrationTables>() {
```

Comment:
If this is going to be a common pattern, we could create a DoFnWLog that puts the logging stuff in the startBundle and finishBundle.